recipes/core/util-linux.yaml

```diff
@@ -34,11 +34,19 @@ buildScript: |
         sbindir=/usr/sbin
 
 multiPackage:
+    "":
+        depends:
+            - name: core::util-linux-tgt
+              use: []
+        packageScript: autotoolsPackageBin
+        provideDeps: [ "*-tgt" ]
+
     dev:
         packageScript: autotoolsPackageDev
+        provideDeps: [ "*-dev" ]
 
     tgt:
-        packageScript: autotoolsPackageTgt
+        packageScript: autotoolsPackageLib
         provideDeps: [ "*-tgt" ]
 
     tools:
```

Comment:
Actually we also could remove the extra tools target and move this into the generic bin target. What do you think?

Reply:
Yes, this sounds reasonable. :+1:

Reply:
done